app/pages/profile/stats.cjsx

```diff
@@ -3,5 +3,5 @@ React = require 'react'
 
 module.exports = (props) =>
   <div className="content-container">
-    <p>Your classification stats are now displayed on <Link to="/">your Zooniverse home page</Link>.</p>
-  </div>
\ No newline at end of file
+    <p>Your classification stats are now displayed on <Link to="https://www.zooniverse.org">your Zooniverse home page</Link>.</p>
+  </div>
```

Comment:
This component is a fallback message for the Stats button on your profile page (e.g. https://www.zooniverse.org/users/[login]/stats prior to Sept 17 launch of new pages). Likely to be forever ignored after this PR is deployed, but being thorough 💁‍♀️